multi-agent/lib/mas/elements/providers/rag_client/client.py

```
@@ -43,11 +43,14 @@ class RagClient:
     DOCS_ENDPOINT = "/api/docs/available.docs.get"
     QUERY_ENDPOINT = "/api/docs/query.match"
 
+    _AUTH_HEADER = "X-Authenticated-User"
+
     def __init__(
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)